apps/emqx_bridge_mqtt/etc/emqx_bridge_mqtt.conf

```diff
@@ -9,8 +9,8 @@
 ## Bridge address: node name for local bridge, host:port for remote.
 ##
 ## Value: String
-## Example: emqx@127.0.0.1,  127.0.0.1:1883
-bridge.mqtt.aws.address = 127.0.0.1:1883
+## Example: emqx@127.0.0.1,  "127.0.0.1:1883"
```

Comment:
this example seem odd from the source branch.

Reply:
ah, ok, node names are allowed. quote should be around both names.
i.e. the field is a string, not a string array. ?